src/components/settings/SettingsSSOTab/tests/ExistingSSOConfigs.test.jsx

```diff
@@ -104,12 +109,15 @@ describe('<ExistingSSOConfigs />', () => {
     );
     expect(screen.getByText('nacho cheese')).toBeInTheDocument();
     expect(screen.getByText('Inactive')).toBeInTheDocument();
-
-    userEvent.click(screen.getByTestId(`existing-sso-config-card-dropdown-${inactiveConfig[0].id}`));
+    act(() => {
+      userEvent.click(screen.getByTestId(`existing-sso-config-card-dropdown-${inactiveConfig[0].id}`));
+    });
```

Comment:
Why is act() needed here?  It was my understanding that @testing-library/react `userEvent` operations are already wrapped in act().

Reply:
I was just following and fixing the warnings printed out by the tests, I believe it has to do with updates triggered by the click and not just the click itself? However I'm not entirely sure